### DISCUSSION TITLE
quick search current WORD like '*' in neovim

### DISCUSSION THREAD

**Discussion by vincentaxhe** [helix-editor/helix#11986] [Q&A] [answered]
I presume if helix do it need to select current WORD first, but then '*' or '/' did not make use of the selection.

**Comment by TornaxO7** [comment, marked as answer]
Well it work on my machine.
For example for the following text:

```
Amogus

A random sentence

Amogus2
```

1. Put your cursor on the first line but also on the first column.
2. Press `e` (in normal mode)
3. Press `*`
4. Press `n`

**Reply by vincentaxhe** [reply]
oh, it did work. the info 'register '/' set to 'xxx'' fooled me.

**Reply by TornaxO7** [reply]
Could you please mark my answer as "answered" then?

![peepoShy](https://github.com/user-attachments/assets/1af5fd1a-8bf2-4747-86d6-b14fc37a50da)

**Reply by ondrej-ivanko** [reply]
> Well it work on my machine. For example for the following text:
> 
> ```
> Amogus
> 
> A random sentence
> 
> Amogus2
> ```
> 
>     1. Put your cursor on the first line but also on the first column.
> 
>     2. Press `e` (in normal mode)
> 
>     3. Press `*`
> 
>     4. Press `n`

This is probably working on stable. When built from bleeding edge(master), than you need to add `*` after selected WORD.
